### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by ANTOINE1128**
# in this PR I have followed the following requirements :
## Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.
## In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    case is_student
+    when 1
+      puts 'Has parent permission? [Y/N]'
+      gets.chomp.downcase
+      student = Student.new(age: age, name: person_name)
+      @persons.push(student)
+    when 2
+      puts 'Specialization'
+      specialization = gets.chomp
+      teacher = Teacher.new(age: age, specialization: specialization, name: person_name)
```
Comment (zieeco) [inline app.rb:53]: - The same error is applicable here also.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    puts 'Age'
+    age = gets.chomp.to_i
+    puts 'Name'
+    person_name = gets.chomp
+
+    case is_student
+    when 1
+      puts 'Has parent permission? [Y/N]'
+      gets.chomp.downcase
+      student = Student.new(age: age, name: person_name)
```
Comment (zieeco) [inline app.rb:48]: - When trying to create a teacher, I get this error. Check your student class in your initialize to see how you've added the parameters to fix this.

image

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,106 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'rental'
+# require_relative 'classroom'
```
Comment (zieeco) [inline app.rb]: - Lett's remove this commented line to keep our code clean

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+require_relative 'teacher'
+# require 'pry'
```
Comment (zieeco) [inline app.rb]: - Lett's remove this commented line to keep our code clean

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ANTOINE1128,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Nice adding the console app :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions


- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 3 above)
Comment (Ismailco) [inline app.rb]: - [ ] Kindly make sure to remove all the commented code from your code base so it looks clean and easy to navigate, as for now lines `5` and `7` are commented

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ANTOINE1128, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ANTOINE1128 

Your project is complete! Great job 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
-  correct gitflow
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**